### PR TITLE
Megan's copy edits

### PR DESCRIPTION
**Pull request by mthibos**
Megan makes copy edits!  :+1: 

## Additions
 - KBYO disclaimer to Journey pages 

## Changes
 - hero copy edits
 - landing page + form landing page copy edits

## Review
@cfarm @stephanieosan 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/process/_process-step.html [hunk 1]
```diff
@@ -180,6 +180,8 @@ <h1 class="content-l_col-1-4 content-l_col h6 u-mb10">
                 </div>
                 <p class="short-desc block block__bg block__sub block__bg-small">
                   Visit our <a href="/owning-a-home/process/sources/">sources page</a> to learn more about the facts and numbers we reference.
+                 <br>
+                 <i>Note: The process and forms described on this page reflect <a href="/know-before-you-owe/" target="_blank">new mortgage regulations</a> that take effect for most mortgages on October 3, 2015.</i>
                 </p>
```
Comment (virginiacc) [inline src/process/_process-step.html]: If you want more spacing between the sources link & note, you could put them in separate paragraphs & add a parent element to apply the background:

```html
<aside class="short-desc block block__bg block__sub block__bg-small">
  <p>
    Visit our <a href="/owning-a-home/process/sources/">sources page</a> to learn more about the facts 
and numbers we reference.
  </p>
  <p class="u-mb0">
    <i>Note: The process and forms described on this page reflect <a href="/know-before-you-owe/" 
target="_blank">new mortgage regulations</a> that take effect for most mortgages on October 3, 2015.</i>
  </p>
</aside>
```

**Comment by virginiacc** [comment]
@mthibos Awesome! :sparkles: 

I had one optional suggestion if you wanted more spacing between the sources link & note on the journey pages, but other than that this is ready to merge!

**Comment by mthibos** [comment]
@virginiacc is the screenshot you posted a reflection of my original code, or your code suggestion?  I don't have a local environment, so there's no way for me to see how things look until they are merged.

**Comment by mthibos** [comment]
@virginiacc   Also, I notice there is a little red X on my next-to-last commit, is that anything I need to worry about?

**Comment by virginiacc** [comment]
@mthibos Sorry, that's what the my suggestion would look like. This is the original:

screen shot 2015-08-27 at 12 06 52 pm

I'm not sure why Travis was showing an error on that commit, but all the checks are passing now so everything should be fine.

**Comment by mthibos** [comment]
@virginiacc  Hmmm yes yours looks way better!  I copy-pasted your code exactly and re-committed it, so it should be good!

**Comment by virginiacc** [comment]
Looks good! :+1:

**Pull request merged**